Directory.Build.props

```diff
@@ -0,0 +1,22 @@
+<Project>
+
+  <PropertyGroup>
+    <Version>1.0.0-alpha.1</Version>
+    <LangVersion>latest</LangVersion>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+<!--<AssemblyOriginatorKeyFile>../../.assets/Sentry.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>-->
```

Comment:
You're better off not strong singing this because Xamarin bits are not strong signed

```suggestion
```